cli/template/base/next.config.mjs

```diff
@@ -1,5 +1,7 @@
 // @ts-check
-import { env } from "./src/env/server.mjs";
+process.env.NODE_ENV === "development"
+  ? await import("./src/env/server.mjs")
+  : null;
```

Comment:
Not a fan of this, if you're not using docker, you'd likely want the build to fail if the envs aren't validated. Either add this snippet to the docker docs, or use another env var that could cancel it?

Reply:
Very true, wasn't thinking of that. I'll think how can that be improved.

Reply:
with another var I mean something like

```ts
/* run the build with this set to skip validation */
!process.env.SKIP_ENV_VALIDATION
  && await import("./src/env/server.mjs");
```

and then in the docker-compose you can build with this set

Reply:
Yep, this seems fine. Thanks!